### ISSUE TITLE
FR: user defined "Quick View" menu option as part of the main menu linup.

### ISSUE THREAD

**Issue by solarkite** [koreader/koreader#8653]
A new main menu option populated by user generated selections from any other menu options. i.e. WiFi toggle, night mode, font selection, front light, profile activation, etc. 
The menu would allow quick monitoring of activities of the Users choice such as WIFI status, keep alive, auto frontlight, etc. while also providing the activation or deactivation of a corresponding element/s.
Implementation, as pointed out by @Frenzie (https://github.com/koreader/koreader/pull/2601#issuecomment-1008371938) could be facilitated by dispatcher. But monitoring of status states would need to be solved, among other challenges.

**Comment by shuvashish76** [comment]
Old issues like this are piling up...
@offset-torque please close this.

**Comment by offset-torque** [comment]
I don't close issues **shuvashish76**. I generally see @Frenzie closing the stale/completed ones and he is also labeling them when closing, so pinging him is the correct action in these cases.

**Comment by Frenzie** [comment]
Assistance with tagging/closing is definitely welcome. :-)

But I'm not really sure where the close request is coming from in this case. Is it about QuickMenu?

**Comment by shuvashish76** [comment]
Yes this issue was created in Jan 11, 2022 then few days after Jan 21, 2022 **QuickMenu** FR #8701 by **offset-torque**.

> A new main menu option populated by user generated selections from any other menu options. i.e. WiFi toggle, night mode, font selection, front light, profile activation, etc.
The menu would allow quick monitoring of activities of the Users choice such as WIFI status, keep alive, auto frontlight, etc. while also providing the activation or deactivation of a corresponding element/s.

This feature sounds exactly what QuickMenu does 
- [x] user generated
- [x] multiple menu options, _while also providing the activation or deactivation of a corresponding element/s_
- [ ] monitoring of activities (status of selected activities? If notification enabled it can show you the status when changed.)

**Comment by Frenzie** [comment]
I see that final aspect as the main request, but I think if you're worried that wifi is on you can just turn it off. Doing so twice doesn't hurt anything.

Regarding the close request, it doesn't need that much detail but I'd suggest writing someone more along the lines of "This can be closed because we now have QuickMenu". Otherwise it's cryptic for anyone who comes across it, and as written here it also seems unnecessarily rude to the OP.
